template_test.go

```diff
@@ -2,13 +2,77 @@ package main
 
 import (
 	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
```

Comment:
FWIW, assert's "MIT license" *is* compatible with goenvtemplator's LGPL3.

Reply:
THX for checking!
![cr-secutiry](https://user-images.githubusercontent.com/6112562/129886523-6219f1c4-494f-4393-98fa-4e823703a8a6.jpg)